MTM2/v2/_init.yaml

```diff
@@ -19,6 +19,9 @@ configuration_file: "Configurable_File_Description_20180831T091922_M2_optical.cs
 # 0-based hardpoints. The index should be from low to high.
 hardpoints: [5, 15, 25, 73, 75, 77]
 
+# Enable the calculation of the look-up table (LUT) for the temperature or not.
```

Comment:
probably better:
```
# Use temperature LUT to compensate mirror deformations caused by temperature gradients
use_temperature_lut: False
```

the problem with the current description is that it suggests you calculate (=fit) the LUT in the code. You don't fit it, you just use it.

Reply:
Improved in the commit: `[8c67b67](https://github.com/lsst-ts/ts_config_mttcs/pull/91/commits/8c67b677345777b319469d4cd4aecd9bd789cb32)`. Thanks!